bitrise/util.go

```diff
@@ -213,11 +213,11 @@ func generateYAML(v interface{}) ([]byte, error) {
 	return bytes, nil
 }
 
-func normalizeValidateFillMissingDefaults(bitriseData *models.BitriseDataModel) ([]string, error) {
+func normalizeValidateFillMissingDefaults(bitriseData *models.BitriseDataModel, secrets ...[]envmanModels.EnvironmentItemModel) ([]string, error) {
```

Comment:
pls use envmanModels.EnvironmentItemModel variadict instead of []envmanModels.EnvironmentItemModel